pyam/core.py

```diff
@@ -169,6 +169,16 @@ def empty(self):
         """Indicator whether ``IamDataFrame`` is empty"""
         return self.data.empty
 
+    def equals(self, other):
+        """Test if two objects contain the same data and meta indicators"""
```

Comment:
this docstring should state that the object must be of type IamDataFrame (or subclasses thereof)

Reply:
done